content/en/docs/concepts/extend-kubernetes/api-extension/custom-resources.md

```diff
@@ -36,10 +36,12 @@ When you combine a custom resource with a *custom controller*, custom resources
 provide a true _declarative API_.
 
 A [declarative API](/docs/concepts/overview/kubernetes-api/)
-allows you to _declare_ or specify the desired state of your resource and tries to
-keep the current state of Kubernetes objects in sync with the desired state.
-The controller interprets the structured data as a record of the user's
-desired state, and continually maintains this state.
+encourages you to *declare* the desired state of your resource.  This is in
+contrast to an imperative API, where you *instruct* a server what to do. The
+Kubernetes declarative API model enforces a separation of responsibilities.
+You, the Kubernetes user, are responsible for declaring the desired state of
```

Comment:
I don't think you need to clarify that the reader is the user. 

Maybe a reword to "When you use a declarative API to specify the desired state of your resource, the controller attempts to keep the current state of Kubernetes objects aligned with your declared desired state."

Reply:
@shannonxtreme I guess what I'm trying to get at here is that using a declarative API doesn't actually have anything to do with a controller reconciling the desired state with the latest observed state. The Kubernetes controller system (and its control loops) is the thing that reconciles observed state to desired state. It's about the division of responsibilities between a user and a controller that is important here.

Reply:
Be careful with “user” there. Taking ReplicaSet as an example, usually the thing that defines a ReplicaSet is&hellip; a different controller.

Reply:
@sftim perhaps "The Kubernetes declarative API model enforces a separation of responsibilities. A controller is responsible for keeping the current state of Kubernetes objects in sync with the desired state of those objects that has been declared by another actor in the system."?

Reply:
@sftim ping :) what do you think of my wording above?

Reply:
SGTM

Reply:
@jaypipes could you please commit this change you've suggested 🙂

Reply:
@RinkiyaKeDad yep, sorry I was on PTO and catching up on $work stuff! :) Will amend shortly.

Reply:
@sftim @RinkiyaKeDad k, updated. thanks for your patience! :)